AirCasting/SessionViews/StandaloneSessionCardView.swift

```diff
@@ -33,6 +33,7 @@ struct StandaloneSessionCardView: View {
         SessionHeaderView(
             action: {},
             isExpandButtonNeeded: false,
+            isMenuNeeded: false,
```

Comment:
@jankrzempek here I'm setting it to false

Reply:
ooh, sorry then. Approve right away!